### PR TITLE
Add option for deploy keys in pack install

### PR DESCRIPTION
**Pull request by danielhneville**

### CODE REVIEW

**Comment by punkrokk** [comment]
I would be interested in getting this over the line of someone could review it.

File: contrib/packs/actions/pack_mgmt/download.py [hunk 1]
```diff
@@ -63,15 +63,15 @@ def __init__(self, config=None, action_service=None):
...
+    def run(self, packs, abs_repo_base, verifyssl=True, force=False, deploykey=None):
         result = {}
 
         for pack in packs:
             pack_result = download_pack(pack=pack, abs_repo_base=abs_repo_base,
                                         verify_ssl=verifyssl, force=force,
                                         proxy_config=self.proxy_config,
                                         force_permissions=True,
-                                        logger=self.logger)
+                                        logger=self.logger, deploy_key = deploykey)
```
Comment (raghavendrapj) [inline contrib/packs/actions/pack_mgmt/download.py]: remove spaces around assignment

**Comment by cognifloyd** [comment]
@danielhneville you may have signed the CLA previously, but we recently updated it to mention the Linux Foundation instead of the previous afterwards of StackStorm. If you can sign the CLA again, then I'll help you rebase this and get it merged.

**Comment by cognifloyd** [comment]
> If I haven't heard back by the end of October 2021, then I will close this PR.

Well, I haven't heard back. If you still want to see this in, please submit a new PR. Thanks!

**Pull request closed without merging**